### DISCUSSION TITLE
Keyboard Light "GPU mode" doesn't work

### DISCUSSION THREAD

**Discussion by MC-dusk** [seerge/g-helper#1912] [General]
When toggle the mode to `GPU mode`, it will keep the display mode before no matter what it is. If just talk about stable release version, v0.137 can work, but v0.140 and after can't.

Here's some info:

ASUS TUF Gaming A15 FA507XV
Win11 22H2
ASUS System Control Interface v3 3.1.22.0 (not latest)
BIOS 312 (not latest)

Should I update the two above?

**Comment by seerge** [comment]
@MC-dusk absolutely nothing has changed in GPU switch in the app for last 10 months probably.

Please post correct Issue along with template and all required info, instead of creating topics in Discussions

Here I won't be able even to look at your case due total lack of information

https://github.com/seerge/g-helper/issues

**Reply by MC-dusk** [reply]
To be honest, I just wanted to see if anyone had a similar situation, so I opened a discussion rather than an issue. I'm not sure if it's my fault or it's a common problem. Also this won't have a significant impact cause I almost only use the `heatmap` mode.

**Comment by seerge** [comment]
@MC-dusk I have checked something, try this build

[GHelper.zip](https://github.com/seerge/g-helper/files/13928783/GHelper.zip)

**Reply by MC-dusk** [reply]
Thank you. This can work normally.

**Reply by seerge** [reply]
Great
